docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -73,6 +73,8 @@ The solution is located in the `getting-started` directory.
 
 The easiest way to create a new Quarkus project is to open a terminal and run the following command:
 
+For linux & mac
```

Comment:
```suggestion
For Linux and MacOS users
```

Reply:
If we want to be pedantic, it should be `macOS` :)

Reply:
I have done with cap "M" for macOS